docs/fides/docs/schemas/config_schema.json

```diff
@@ -29,7 +29,8 @@
       "title": "Cli",
       "default": {
         "local_mode": false,
-        "server_url": "http://localhost:8080"
+        "server_url": "http://localhost:8080",
+        "analytics_id": "6044ce70af305cf78969a3cd7a5371ba"
```

Comment:
Please remove this line. Even though an `analytics_id` is required (for clients that are opted into analytics), this value is associated with a specific instance and should not exist in VCS.